lib/adapters/REST/endpoints/asset.ts

```diff
@@ -204,7 +204,7 @@ export const createFromFiles: RestEndpoint<'Asset', 'createFromFiles'> = (
 const ASSET_PROCESSING_CHECK_WAIT = 3000
 const ASSET_PROCESSING_CHECK_RETRIES = 10
 
-function checkIfAssetHasUrl(
+async function checkIfAssetHasUrl(
```

Comment:
Why this change (and in some of the other functions) considering there is no `await`?